RecoTracker/TkNavigation/plugins/NavigationSchoolESProducer.cc

```diff
@@ -58,13 +59,17 @@ NavigationSchoolESProducer::ReturnType NavigationSchoolESProducer::produce(const
   using namespace edm::es;
 
   //get the geometricsearch tracker geometry
+  if (navigationSchoolPluginName_ == "")
+    navigationSchoolPluginName_ = navigationSchoolName_;
+  const std::string pluginName_(navigationSchoolPluginName_);
```

Comment:
This could be done already in the constructor.

Reply:
Done in the last update.